staging/src/kubesphere.io/api/cluster/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -16,7 +16,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Code generated by deepcopy-gen. DO NOT EDIT.
+// Code generated by controller-gen. DO NOT EDIT.
```

Comment:
Please use deepcopy-gen, it's recommended. Using different generators with cause overwriting generated code files again and again. Maybe we should file an issue to upstream, suggest align `deepcopy-gen` behaviors with `controller-gen`

Reply:
This was generated by`make manifests` command. the `Makefile` should be fixed?

Besides, I think all  the generated file should be processed during `CI`.

Reply:
Yeah, that's the place we need to improve. Actually, `make target` only invoke the local go package, it didn't check which version you are using. Please undo the code change first, we'll submit another PR to deal with it.

Reply:
ok，I'll redo this pr after this fix merged

Reply:
What I mean is, checkout `staging/src/kubesphere.io/api/cluster/v1alpha1/zz_generated.deepcopy.go` alone, since it's basically the same the older version.

Reply:
ok,  then your PR will process this code gen in CI?

Reply:
and how about `crds/cluster.kubesphere.io_clusters.yaml` ?

Reply:
keep the change to `crds/cluster.kubesphere.io_clusters.yaml`, that's fine.

Reply:
updated